Contributing/How-to-Contribute-Land-Sector-Data-Analysis.md

```diff
@@ -0,0 +1,441 @@
+>These datasets have been used by many countries and are effective in real-world scenarios. For each dataset information is provided on content and license. Proper permissions have been cited for all the datasets and we urge you to follow the license for every dataset before proceeding to work with it.
+
+If you have another dataset on your mind that would be beneficial to [Moja Global's mission](https://moja.global/), please feel free to reach out to us on [info@moja.global](info@moja.global) to share details about this dataset.
+
+
+Concerning the data analysis software or programming language for the analysis, we encourage you to pick a softwate that supports geospatial analysis since you will be dealing with polygon files. 
+Moja Global is an open source project, the interest here is to share your processes(**code**) and results(**interpretations**) to support the open source movement. In order to efficiently doucment your analysis, we advice you use [notebooks](https://sourceforge.net/software/product/Google-Colab/alternatives). Notebooks are documents produced by the data analysis software, which contain both computer code (e.g. python) and rich text elements (paragraph, equations, figures, links, etc…). 
+
+Notebook documents are both human-readable documents containing the analysis description and the results (figures, tables, etc..) as well as executable documents which can be run to perform data analysis. 
+
+It is important to note that when you have your contribution ready, you would have to make PR to have it merged. Please refer to the [community website](https://github.com/moja-global/About_moja_global/blob/master/Contributing/How-to-Contribute-Code.md) to know how to make a pull request.
+
+---
+
+
+
+If you have experience and want to help, please send an email to [mentorship@moja.global](mailto:mentorship@moja.global). We are open to mentoring new developers and contributors through various mentorship program and initiatives.
+
+
+
+
+With this in mind, we will be expecting different notebooks which is good since it would be at your convinience and you can reveal insight which has not been previously uncovered.
+
+
+When we look at climate action, we would like to get descriptions from a global scale to a specific state administration wthin a particular country. A draft review of the characteristics of each file in the dataset is found [here](https://colab.research.google.com/github/Simpleshell3/Land-Sector-Dataset-Description/blob/main/LandSectorDatasetDescription101.ipynb).
+
+
+Amongst the numerous factors that affecct climate change, the Agriculture, Forestry and Other Land Use (AFOLU) sector is responsible for almost a quarter of the global Greenhouse gases (GHG) emissions. The emissions associated with AFOLU activities are projected to increase in the future.
+
+
+The following sections of this guide will direct you on how to document your analysis at any scale(global, continental, country, state).
+

```

Comment:
I think the introduction can be put into subsections:

The dataset, data analysis, notebook, and contributing. [More or less sections if I have missed out on anything.]

Reply:
Ohh great thank you, I have added the [subsections](https://github.com/Simpleshell3/About_moja_global/blob/master/Contributing/How-to-Contribute-Land-Sector-Data-Analysis.md#10-introduction) to the guide.

Reply:
Hey that looks great.


Maybe you could take help of the notes on technical documentation... Since, there are a few documentation guidelines that are to be followed.

Just make the changes commit the changes to this PR.

It will take 2-3 rounds of review and changes but you're doing really well.

Reply:
Hello @chicken-biryani . I have made some changes, please review it for me.